plugins/feeds/public/cisco_umbrella_top_domains.py

```diff
@@ -0,0 +1,39 @@
+import io
+from datetime import timedelta
+from typing import ClassVar
+
+from core import taskmanager
+from core.config.config import yeti_config
+from core.schemas import task
+from core.schemas.observables import hostname
+
+
+class CiscoUmbrellaTopDomains(task.FeedTask):
+    _defaults = {
+        "frequency": timedelta(hours=1),
```

Comment:
```suggestion
        "frequency": timedelta(hours=24),
```

24 hours (or even more?) is probably enough, no?

Reply:
Fixed in e5e40e9b30016c5ac14f85450cc9862ebf2b0095